tests/mocha/workspace_test.js

```diff
@@ -7,6 +7,18 @@
 suite('Workspace', function() {
   setup(function() {
     this.workspace = new Blockly.Workspace();
+  });
+
+  teardown(function() {
+    this.workspace.dispose();
+  });
+
+  // eslint-disable-next-line no-use-before-define
```

Comment:
Why did you move the test definition out into a separate function? It's obviously intentional but I don't understand the benefit.

Reply:
I wouldn't be able to access the test in workspace_svg_test if I defined it within the workspace test suite.